x-pack/plugins/triggers_actions_ui/public/application/components/add_message_variables.tsx

```diff
@@ -27,44 +33,96 @@ interface Props {
   showButtonTitle?: boolean;
 }
 
+const LOADING_VARIABLES = i18n.translate(
```

Comment:
nit -> might be nice to create a `translations.ts` file to store all of these constants.

Reply:
1e78806f0c65364af270f322b05f529bb20097d5